Dockerfile

```diff
@@ -6,12 +6,16 @@ WORKDIR /src
 RUN npm install
 ARG viewer
 ARG fork
-RUN git clone https://github.com/${fork:-camicroscope}/camicroscope.git --branch=${viewer:-master}
+# RUN git clone https://github.com/${fork:-camicroscope}/camicroscope.git --branch=${viewer:-master}
```

Comment:
@Vedant1202 just remember to un-comment it again before we merge :)

Reply:
done